src/sentry/migrations/0300_backfill_msteams_default_installation_type.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.28 on 2022-07-11 18:13
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_msteams_installation_type(apps, schema_editor):
+    Integration = apps.get_model("sentry", "Integration")
+
+    for integration in RangeQuerySetWrapperWithProgressBar(
+        Integration.objects.filter(provider="msteams")
```

Comment:
There are ~3k rows that match this criteria. You should be able to run this during the deploy window, but if you want to be safe you could make this migration `is_dangerous`.